test/e2e/suite/carotation/carotation.go

```diff
@@ -135,30 +138,40 @@ var _ = framework.CasesDescribe("CA rotation", func() {
 		caData, ok := caSecret.Data["ca.crt"]
 		Expect(ok).To(BeTrue(), "expected 'ca.crt' to be present in Issuer CA Secret")
 
+		tlsData, ok := caSecret.Data["tls.crt"]
+		Expect(ok).To(BeTrue(), "expected 'tls.crt' to be present in Issuer CA Secret")
+
+		Expect(caData).To(Equal(tlsData), "invalid test; expected 'ca.crt' to equal 'tls.crt' for Issuer CA Secret (this implies a previous test run wasn't cleaned up correctly)")
+
 		pod1Bundle, err := util.ReadCertFromMountPath(f, mountPath, pod1.Name, containerName)
 		Expect(err).NotTo(HaveOccurred())
 
 		pod2Bundle, err := util.ReadCertFromMountPath(f, mountPath, pod2.Name, containerName)
 		Expect(err).NotTo(HaveOccurred())
 
-		Expect(caData).To(Equal(pod1Bundle.CAPEM), "expected Issuer CA bundle to equal CA mounted in pod1 file")
+		Expect(string(caData)).To(Equal(string(pod1Bundle.CAPEM)), "expected Issuer CA bundle to equal CA mounted in pod1 file")
 
-		Expect(caData).To(Equal(pod2Bundle.CAPEM), "expected Issuer CA bundle to equal CA mounted in pod2 file")
+		Expect(string(caData)).To(Equal(string(pod2Bundle.CAPEM)), "expected Issuer CA bundle to equal CA mounted in pod2 file")
```

Comment:
note: comparing strings makes it easier to visually see what went wrong when the objects under test are not equal.